### DISCUSSION TITLE
cannot read property animations of null

### DISCUSSION THREAD

**Discussion by hilarytsai** [mozilla/hubs#3416] [Spoke Questions]
I'm unable to publish my scene because of an error "cannot read property animations of null."  I don't have any nulls in my project, and this has never happened before. All assets animations run smoothly when I hit preview.  Is this a known issue, or something likely on my end?

**Comment by cvanmeurs** [comment]
Check to make sure you don't have an object in your scene that has a loop-animation component on it and no animation selected from the drop-down menu.  The "null" is probably referring to a missing assignment and not a null object.

**Comment by pattersonbl2** [comment]
are you still having this issue ? if so let me know
